code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -175,38 +175,85 @@
 /mob/living/carbon/human/proc/commune()
 	set category = "Abilities"
 	set name = "Commune with creature"
-	set desc = "Send a telepathic message to an unlucky recipient."
+	set desc = "Send a telepathic message to a recipient."
+
+	var/obj/item/organ/external/rhand = src.get_organ("r_hand")
+	var/obj/item/organ/external/lhand = src.get_organ("l_hand")
+	if((!rhand || !rhand.is_usable()) && (!lhand || !lhand.is_usable()))
+		to_chat(src,"<span class='warning'>You can't communicate without the ability to use your hands!</span>")
+		return FALSE
+	if((lhand.is_stump()) && (rhand.is_stump()))
+		to_chat(src,"<span class='warning'>You can't communicate without functioning hands!</span>")
+		return FALSE
+	if(src.r_hand != null && src.l_hand != null)
```

Comment:
This is also handled in line 182.

Reply:
No, this is to check if your hands are full.

Reply:
Oh, fuck, I'm blind. Yes, there are underscores there.